docs/install_guides/pypi-openvino-rt.md

```diff
@@ -25,7 +25,9 @@ The Intel® Distribution of OpenVINO™ toolkit for Linux\*:
 * Supported Operating Systems:
   - Ubuntu* 18.04 long-term support (LTS), 64-bit (python 3.6 or 3.7)
   - Ubuntu* 20.04 long-term support (LTS), 64-bit (python 3.6 or 3.7)
-  - macOS* 10.15.x version
+  - Red Hat* Enterprise Linux* 8.2 (64 bit) (python 3.6 or 3.7)
+  - CentOS 7.4 (64 bit) (python 3.6 or 3.7)
+  - macOS* 10.15.x versions
   - Windows 10*, 64-bit Pro, Enterprise or Education (1607 Anniversary Update, Build 14393 or higher) editions
```

Comment:
Seems to me that providing Python versions matrix for Mac and Windows is a good idea.

Reply:
Updated, let's review how it looks now